library/tiqr/Tiqr/HealthCheck/Interface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+interface Tiqr_HealthCheck_Interface
+{
+    /**
+     * Do a health check
+     *
+     * @param string &$statusMessage: a message that describes the status of the health check.
+     *
+     * This message is set by the health check when it fails to provide more information about the failure.
+     * This message should be a short string that can be displayed to a user or logged.
+     * It must not contain any sensitive information.
```

Comment:
I understand the usefulness of this construction. In my opinion this is not how we should retrieve the statusMessage information. Having a method return multiple 'outputs' is usually not considered good practice in our neck of the woods.

In this case the `healthCheck` method returns true or false, indicating if the healthcheck was a success or not. But also 'returns' a status message.

I'd personally just add a `getStatusMessage` method to the health checker. The content of that message can be set by the healthCheck method.